benchmarks/profiler/utils/config.py

```diff
@@ -107,7 +108,7 @@ class DgdPlannerServiceConfig(BaseModel):
     extraPodSpec: PodSpec = PodSpec(
         mainContainer=Container(
             image="my-registry/dynamo-runtime:my-tag",  # placeholder
-            workingDir="/workspace/components/src/dynamo/planner",
+            workingDir=f"{get_workspace_dir()}/components/src/dynamo/planner",
```

Comment:
There might be a small issue here if the profiler's image (where this python code is running in) has a different workspace dir than workspace dir of the image where this generated DGD is using. 
With that said, the current approach is no better (and it fails for sgl-wideep image 100% sure). 

```suggestion
            # todo: get the accurate workspace dir in the docker image instead of the workspace dir of the profiler image
            workingDir=f"{get_workspace_dir()}/components/src/dynamo/planner", 
```